news/2022-05-21-project-loved-may-2022.md

```diff
@@ -0,0 +1,321 @@
+
+As a featured map in the [MWC 2020 quarterfinals](/wiki/Tournaments/MWC/2020_4K), this would add to the mid-to-high level of density oriented long note catalogue of the Loved section. Speaking of which, this is not Raveille's first long note Loved entry, and some of his prouder charts like [Delrio](https://osu.ppy.sh/beatmapsets/1251408) or [Mare Maris](https://osu.ppy.sh/beatmapsets/917906) have even gone as far as getting ranked! The popularity aspect of *Bi*, however, had us interested in including it in the Loved section.
+
+Have you decided on your favorite letter of the alphabet yet? Feel free to tell us about it in the voting thread, and don't forget to cast your vote as well!
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/789869.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Camellia - liquated](https://osu.ppy.sh/beatmapsets/789869#mania) by [-MysticEyes](https://osu.ppy.sh/users/6253266)\
+140 BPM, 4:22 | 4.65★\
+*description written by [Paturages](https://osu.ppy.sh/users/1375479)*
+
+For a decent amount of time, -MysticEyes has been a reliable source of maps of both convincing quality and adventurous ambition. With no less than 15 ranked mapsets and [one Loved LN entry](https://osu.ppy.sh/beatmapsets/800341), -MysticEyes has experimented and played around with a lot of different mapping styles and skillset focuses.
+
+As such, *liquated* is a surprising entry. Its focus resides in something that has been a bit popular lately in the ranking section: stream dumps. You would not expect to find such a map in -MysticEyes's repertoire, especially not in 2018. At that point in time, it would have been less imaginable to rank this type of map, and as much as we would like to see this resurface there in 2022, this map will have to remain a testament of the past.
+
+Despite not being featured in any major way and being more than 4 minutes long, it has nonetheless amassed a fair amount of spontaneous popularity, enough for us to consider this for the Loved section. Give this one a try: you will be delightfully surprised, as this plays quite smoothly throughout all its length. Don't forget to hit us with a vote afterwards!
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1428709.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[C-show - GIMME DA BLOOD](https://osu.ppy.sh/beatmapsets/1428709#mania) by [H1Pur](https://osu.ppy.sh/users/15756120)\
+220 BPM, 2:11 | 2.75★, 2.75★\
+*description written by [Paturages](https://osu.ppy.sh/users/1375479)*
+
+H1Pur finally makes his entrance to the Loved section! As an up and coming mapper who has recently been contributing to a number of tournaments with tasteful SV mapping, starting with [SVSE](https://osu.ppy.sh/community/forums/topics/1313374), he is also perhaps the best known enthusiast of Amelia Watson we know in the osu!mania community. Feel free to prove us wrong in the voting thread.
```

Comment:
```suggestion
H1Pur finally makes his entrance to the Loved section! As an up and coming mapper who has recently been contributing to a number of tournaments with tasteful SV mapping, starting with [SVSE](https://osu.ppy.sh/community/forums/topics/1313374), he is also perhaps the best known enthusiast of the Amelia Watson we know in the osu!mania community. Feel free to prove us wrong in the voting thread.
```

Reply:
"we know in the osu!mania community" refers to H1Pur